flytekit/core/python_function_task.py

```diff
@@ -607,3 +607,32 @@ async def run_with_backend(self, **kwargs):
                 # now have to fail this eager task, because we don't want it to show up as succeeded.
                 raise FlyteNonRecoverableSystemException(base_error)
             return result
+
+    def run(self, remote: "FlyteRemote", ss: SerializationSettings, **kwargs):  # type: ignore[name-defined]
```

Comment:
where/how is this called?

Reply:
see comment... it's not yet, but it's super helpful for running eager locally, but spawning tasks against a live backend.  used in conjunction with this code here: https://github.com/unionai/debugyt/blob/ccd5dc4648bd223fca9f9957a4bc07bff642aab6/user/ytong/src/async_eager/threaded_oss.py#L169  I was going to formalize it (so users can run eager locally but spawn tasks against serverless) but it hasn't gotten any buy in.  so a test function it remains for now.

Reply:
Thanks for explaining, even after reading the comments I was having trouble visualizing how we were going to call it, but the example clarified the intended use.